test/mobile_wall_hug/scenarios/mobilewallhug_footpath2.txt

```diff
@@ -1,7 +1,7 @@
 VERSION(2.300000)
 SET_LAND_NUMBER(0)
 LOAD_LANDSCAPE(".\DATA\LANDSCAPE\MULTI_PLAYER\MPM_2P_1.lnd")
-START_CAMERA_POS("2197.83,2372.96")
+rem START_CAMERA_POS("2197.83,2372.96")
```

Comment:
This is probably because camera is null in game

Test if fdb5bd117b80d3d885902e4abf2bf35d7f15ad01 fixes it.